test-integration/Makefile

```diff
@@ -1,3 +1,3 @@
 .PHONY: test
 test:
-	pytest -s --tb=short
+	pytest -vv
```

Comment:
The purpose of `--tb=short` was so pytest didn't spit out thousands of irrelevant lines of junk, and `-s` was there so I could drop to ipdb if I needed to debug a test. I might bring those back if I run into that again.

Reply:
Perhaps it could be a separate command, or you could just add it if you need it. No output capturing makes debugging without ipdb impossible. ;)